minidump-common/src/format.rs

```diff
@@ -300,6 +300,10 @@ pub enum MINIDUMP_STREAM_TYPE {
     ///
     /// See ['DSO_DEBUG_64'] and ['DSO_DEBUG_32'].
     LinuxDsoDebug = 0x4767000A,
+    /// The number of FDs in /proc/self/fd/ from a Linux system
```

Comment:
Note that the 0x4767 is `Gg` to indicate Google extensions [per the Breakpad source](https://chromium.googlesource.com/breakpad/breakpad/+/f49c2f1a2023da0cb055874fba050563dfea57db/src/google_breakpad/common/minidump_format.h#347).

To reduce the possibility of conflicting with future Google streams I'd recommend numbering this with the `0x4d7a` Mozilla extension streams below.